vllm/v1/spec_decode/eagle.py

```diff
@@ -308,6 +309,9 @@ def load_model(self, target_model: nn.Module) -> None:
         loaded_weights = self.model.load_weights(
             loader.get_all_weights(draft_model_config, self.model))
 
+        process_weights_after_loading(self.model, draft_model_config,
+                                      target_device)
```

Comment:
FWIW, what I propose in #18273 would have the same effect

Reply:
@markmc I've merged this PR before #18273 as the PR has CI errors. Please feel free to fix this in your PR.